configuration/csv-to-docs.md

```diff
@@ -42,4 +286,10 @@ Create the `.gdrive.secrets.json` file by downloading the `client_secrets.json`
 
 ```
 
- See google's docs [here](https://github.com/googleapis/google-api-nodejs-client#oauth2-client) on Oauth2
\ No newline at end of file
+ See Google's docs [here](https://github.com/googleapis/google-api-nodejs-client#oauth2-client) on Oauth2
+
+## Converting CSVs and Uploading Docs
```

Comment:
I feel like this should be before everything else? Isn't this knowledge required before the entire rest of the doc?

Reply:
Addressed in https://github.com/medic/medic-docs/commit/563a2af370f9e3b41d04e6d8905daf1e74adf251